src/Console/ConfigurationResolver.php

```diff
@@ -889,6 +904,9 @@ private function setOption(string $name, $value): void
         $this->options[$name] = $value;
     }
 
+    /**
+     * @param 'allow-risky'|'using-cache' $optionName
```

Comment:
this method was supposed to be generic, regardless which option passed it should not complain, as there is lot of validation for expected type

Reply:
Fixed